src/query/builder.js

```diff
@@ -806,13 +806,36 @@ assign(Builder.prototype, {
   },
 
   // Increments a column's value by the specified amount.
-  increment(column, amount) {
+  increment(column, amount = 1) {
+    if (isObject(column)) {
+      for (const key in column) {
+        this._counter(key, column[key]);
+      }
+
+      return this;
+    }
+
     return this._counter(column, amount);
   },
 
   // Decrements a column's value by the specified amount.
-  decrement(column, amount) {
-    return this._counter(column, amount, '-');
+  decrement(column, amount = 1) {
+    if (isObject(column)) {
+      for (const key in column) {
+        this._counter(key, -column[key]);
+      }
+
+      return this;
+    }
+
+    return this._counter(column, -amount);
+  },
+
+  // Clears increments/decrements
+  clearCounters() {
```

Comment:
Could you explain what this endpoint might be used for?
My reaction is that it feels unnecessary

Reply:
It follows the same principals as `.clearSelect()`, `.clearWhere()`, `.clearOrder()`.

I've yet to find a real life use-case for these functions myself, but for consistency I added one here as well. Suppose someone has found a use for the other clear-functions, else they wouldn't exist.

Reply:
@wubzz 
we use it like this with objection.js for example:

![image](https://user-images.githubusercontent.com/1305378/49872936-d8850080-fe1a-11e8-9a45-82ecd8bf1756.png)